pallets/runtime/develop/build.rs

```diff
@@ -1,9 +1,12 @@
 use wasm_builder_runner::WasmBuilder;
 
+const BUILDER_REPO: &str = "https://github.com/PolymathNetwork/substrate-wasm-builder.git";
+const BUILDER_REV: &str = "b0303c15b662dd768bfe808e3ed47ec65305fe2a";
+
 fn main() {
     WasmBuilder::new()
         .with_current_project()
-        .with_wasm_builder_from_path("../../../utils/wasm-builder")
+        .with_wasm_builder_from_git(BUILDER_REPO, BUILDER_REV)
```

Comment:
What breaks if we use the default wasm-builder without the `no-std` feature?

Reply:
It is required by "crypto-library". It is used to select which **backend** should be used: _u32, u64, or avx2_.
The problem is that those features are **mutually exclusive**, so you cannot enable `u32` by default, and `avx2` on `std`.

When we did the integration, the `target_arch` = wasm32 was not working, so we customized the builder to inject the `no_std` feature. We could/should review if, on the latest updates, cargo/rust supports that target arch and in that case, we could back to the original wasm_builder.

Reply:
What if we put `u32` in `no_std` and `avx2` in `std` feature?

Can you point me to a commit that doesn't compile? I am not sure if I understand the problem fully.

Reply:
I don't have the commit, but you can apply the following patch to see the error:

```
diff --git a/pallets/runtime/develop/build.rs b/pallets/runtime/develop/build.rs
index 19f1903c..736167cf 100644
--- a/pallets/runtime/develop/build.rs
+++ b/pallets/runtime/develop/build.rs
@@ -6,7 +6,8 @@ const BUILDER_REV: &str = "b0303c15b662dd768bfe808e3ed47ec65305fe2a";
 fn main() {
     WasmBuilder::new()
         .with_current_project()
-        .with_wasm_builder_from_git(BUILDER_REPO, BUILDER_REV)
+        .with_wasm_builder_from_crates("2.0.0")
         .export_heap_base()
         .import_memory()
         .build()
```